src/lib/RoomConnection.ts

```diff
@@ -156,6 +159,12 @@ export default class RoomConnection extends TypedEventTarget {
 
         const urls = fromLocation({ host: this.roomUrl.host });
 
+        // Set up local media
```

Comment:
Nit picking here really. My personal preference would be to have a function called `setUpLocalMedia` extracted here rather then the inline comment.

Reply:
Will wait with extracting these until we have some more core 😄